CHANGE_LOG.md

```diff
@@ -30,6 +30,10 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Added
```

Comment:
This should be "Changed" instead of "Added"

Reply:
Fixed